jupyter-docker/Dockerfile

```diff
@@ -30,19 +30,19 @@ ENV HAIL_HOME /hail
 # mount this from outside
 ENV HADOOP_CONF_DIR /etc/hadoop/conf
 
-ENV PATH $SPARK_HOME:$SPARK_HOME/python:$SPARK_HOME/bin:$HAIL_HOME:$PATH
-ENV PYTHONPATH $PYTHONPATH:$HAIL_HOME/$HAILPYTHON:$HAIL_HOME/python:$SPARK_HOME/python:$SPARK_HOME/python/lib/py4j-0.10.3-src.zip
-
 ENV USER jupyter-user
 ENV UID 1000
 ENV HOME /home/$USER
 
 # ensure this matches c.NotebookApp.port in jupyter_notebook_config.py
 ENV JUPYTER_PORT 8000
-ENV JUPYTER_HOME /etc/jupyter/
+ENV JUPYTER_HOME /etc/jupyter
```

Comment:
removing the trailing / makes sure we don't end up with double-slashes when we do `$JUPYTER_HOME/foo`